frame/support/src/traits/tokens/nonfungible.rs

```diff
@@ -65,11 +65,14 @@ pub trait Inspect<AccountId> {
 /// Interface for enumerating items in existence or owned by a given account over a collection
 /// of NFTs.
 pub trait InspectEnumerable<AccountId>: Inspect<AccountId> {
+	type ItemsIter: Iterator<Item = Self::ItemId>;
```

Comment:
What do you think about changing to `type ItemsIterator: Iterator<Item = Self::ItemId>;`?
Not a stopper for me though that version could look more readable

Reply:
I was just following general conventions from std: https://doc.rust-lang.org/stable/std/?search=Iter

but I see that substrate uses Iterator, rather than Iter (https://paritytech.github.io/substrate/master/frame_support/storage/trait.IterableStorageNMap.html#associatedtype.Iterator), so I can change it for consistency :+1:

Reply:
yep, that would be nice. Thanks!